pyvista/themes.py

```diff
@@ -0,0 +1,1999 @@
+        pyvista.
+
+        >>> from pyvista.themes import DarkTheme
+        >>> my_theme = DarkTheme()
+        >>> my_theme.show_edges = True
+        >>> pyvista.global_theme.load_theme(my_theme)
+        >>> pyvista.global_theme.show_edges
+        True
+
+        """
+        if not isinstance(theme, DefaultTheme):
+            raise TypeError('``theme`` must be a pyvista theme like '
+                            '``pyvista.themes.DefaultTheme``')
+
+        for name, value in vars(theme).items():
+            setattr(self, name, value)
+
+    def save(self, filename):
+        """Serialize this theme to a json file.
+
+        Examples
+        --------
+        Export and then load back in a theme.
+
+        >>> import pyvista
+        >>> theme = pyvista.themes.DefaultTheme()
+        >>> theme.background = 'white'
+        >>> theme.save('my_theme.json')  # doctest:+SKIP
+        >>> loaded_theme = pyvista.load_theme('my_theme.json')  # doctest:+SKIP
+        """
+        with open(filename, 'w') as f:
+            json.dump(self.to_dict(), f)
+
+
+class DarkTheme(DefaultTheme):
+    """Dark mode theme.
+
+    Black background, "viridis" colormap, tan meshes, white (hidden) edges.
+
+    Examples
+    --------
+    >>> import pyvista
+    >>> from pyvista import themes
+    >>> pyvista.set_plot_theme(themes.DarkTheme())
+
+    Alternatively, set via a string.
+
+    >>> pyvista.set_plot_theme('dark')
+
+    """
+
+    def __init__(self):
+        """Initialize the theme."""
+        super().__init__()
+        self.name = 'dark'
+        self.background = 'black'
+        self.cmap = 'viridis'
+        self.font.color = 'white'
+        self.show_edges = False
+        self.color = 'tan'
+        self.outline_color = 'white'
+        self.edge_color = 'white'
+        self.axes.x_color = 'tomato'
+        self.axes.y_color = 'seagreen'
+        self.axes.z_color = 'blue'
+
+
+class ParaViewTheme(DefaultTheme):
+    """Set the theme to a paraview-like theme.
+
+    Examples
+    --------
+    >>> import pyvista
+    >>> from pyvista import themes
+    >>> pyvista.set_plot_theme(themes.ParaViewTheme())
+
+    Alternatively, set via a string.
+
+    >>> pyvista.set_plot_theme('paraview')
+
+    """
+
+    def __init__(self):
+        """Initialize theme."""
+        super().__init__()
+        self.name = 'paraview'
+        self.background = tuple(PARAVIEW_BACKGROUND)
+        self.cmap = 'coolwarm'
+        self.font.family = 'arial'
+        self.font.label_size = 16
+        self.font.color = 'white'
+        self.show_edges = False
+        self.color = 'white'
+        self.outline_color = 'white'
+        self.edge_color = 'black'
+        self.axes.x_color = 'tomato'
+        self.axes.y_color = 'gold'
+        self.axes.z_color = 'green'
+
+
+class DocumentTheme(DefaultTheme):
+    """Set the global theme to the document theme.
+
+    This theme uses a white background, the "viridis" colormap,
```

Comment:
```suggestion
    """This theme uses a white background, the "viridis" colormap,
```

Reply:
Our documentation linting tool doesn't like it if the first line of the docstring doesn't end in a period.
```suggestion
    """A document theme well suited for papers and presentations.

    This theme uses a white background, the "viridis" colormap,
```